testcode/lib/conio.c

```diff
@@ -120,6 +132,11 @@ void main(void)
                     cputc(i);
                     inpos = (inpos + 1) & 7;
                 }
+#endif
+/* not all targets have waitvblank() */
```

Comment:
Maybe I'm missing the point but the porgram seems to be intended to run through the loop only once after user interaction. And in that scenario there's just no need for waitvblank() - idenpendetly from the fact if it's available or not.

Wouldn't it be preferable to replace the CONIO input with some joystick input on the targets in question - simliar to hello.c ?